src/uproot/source/file.py

```diff
@@ -69,7 +69,9 @@ def get(self, start, stop):
         return self._file.read(stop - start)
 
     @staticmethod
-    def future(source, start, stop):
+    def future(
+        source: uproot.source.file.MultithreadedFileSource, start: int, stop: int
+    ):
```

Comment:
Is it always a `MultithreadedFileSource`? It might always be some kind of file Source.

```suggestion
    def future(
        source: uproot.source.chunk.Source, start: int, stop: int
    ):
```